OliveVSIX/NugetPacker/NugetPackerLogic.cs

```diff
@@ -127,8 +116,11 @@ static bool TryPush(string packageFilename, out string message)
         static bool TryPackNuget(string nuspecAddress, out string message) =>
             ExecuteNuget($"pack \"{nuspecAddress}\" -OutputDirectory \"{NugetPackagesFolder}\"", out message, NugetExe);
 
-        static bool TryPackDotnet(string projectAddress, out string message) =>
-            ExecuteNuget($"pack \"{projectAddress}\" -o \"{NugetPackagesFolder}\"", out message, "dotnet");
+        static bool TryPackDotnet(string projectAddress, out string message)
+        {
+            ExecuteNuget($"build \"{projectAddress}\"", out message, "dotnet");
```

Comment:
The important part that I missed in my previous push was that I forgot to build the project again, so there was a chance that we push older DLL.